Gruntfile.js

```diff
@@ -1,61 +1,38 @@
 module.exports = function (grunt) {
-    // Setup default urls for the irma web server, and irma api server urls
-    // these are used to configure the server pages (so it can find
+    // Setup default urls for the irma api server url.
+    // These are used to configure the server pages (so it can find
     // the API) and the examples (so they can find the authentication server)
-    var server_url, web_server_url, api_server_url;
+    var server_url, api_server_url;
```

Comment:
Now that the `web_server_url` parameter is gone, the `server_url` should go too (its purpose was to enable the user to provide one base url for the `web_server_url` and `api_server_url`).